doc/api/repl.md

```diff
@@ -78,15 +78,15 @@ The default evaluator supports direct evaluation of JavaScript expressions:
 ```js
 > 1 + 1
 2
-> var m = 2
+> const m = 2
 undefined
 > m + 1
 3
 ```
 
-Unless otherwise scoped within blocks (e.g. `{ ... }`) or functions, variables
-declared either implicitly or using the `var` keyword are declared at the
-`global` scope.
+Unless otherwise scoped within blocks or functions, variables declared
+either implicitly, or using the `const`, `let`, or `var` keywords
```

Comment:
Do you mean "`let`, or `var`"? It is a @lance [sentence](https://github.com/nodejs/node/pull/10160#discussion_r91355653). It seems he uses [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) here. Should I change it to "`let` or `var`"?

Or do you mean another comma?